### DISCUSSION TITLE
Using Relationships Children of Relationship in Lookup

### DISCUSSION THREAD

**Discussion by joaonivaldo** [Power-Components/livewire-powergrid#747] [Q&A]
Hello, I need to do a search on a property that is within a child relationship and I'm not getting it. I need to search the property nomeExibicao_nomeFantasia that is inside the RECORD relationship that is inside the OWNER relationship of my PG model. Like this: $rate->owner->record->nomeExibicao_nomeFantasia.

![image](https://user-images.githubusercontent.com/4040086/205154782-376ba254-30be-494f-9d91-a65bfbddbead.png)

I put the relationSearch like this and it presents the following error:

![image](https://user-images.githubusercontent.com/4040086/205154943-21205713-00ce-439a-9c6b-93c62a162448.png)

See that it is looking for the RECORD relationship within the Model RATE. Since in fact the Record is inside the OWNER which is a relationship of the Rate model.

Thanks

**Comment by BenOussama180** [comment]
can you please show your datasource?

**Comment by joaonivaldo** [comment]
![image](https://user-images.githubusercontent.com/4040086/205777463-ae5bae3c-48fd-4118-a35c-51259062d825.png)

**Reply by BenOussama180** [reply]
i think u need to join the tables as said in the documentation 
https://livewire-powergrid.com/#/table/datasource?id=join-tables
